### ISSUE TITLE
Broken playback through Bluetooth

### ISSUE THREAD

**Issue by matthewmorek** [voltraco/beta#88]
### Issue
When played through Bluetooth devices using built-in iOS output controls, the playback is inconsistent, unresponsive and sometimes does not work at all.

### Impact
Whenever an issue happens, it either stops playback altogether and the app has to be relaunched, or it becomes stuck and plays properly only one or two tracks, then stops playback again.

### Steps to reproduce
1. Connect your iPhone to any Bluetooth speaker (or a car).
2. Play some music (either stream or play downloaded tracks)
3. Try switching between tracks via either Voltra or speaker controls (car controls)
4. After a few skips, Voltra will stop playing music and won’t do it until you relaunch the app.

### Expected result
Using either Voltra or device playback controls should result in smooth skipping/pausing of tracks.
